.vscode/settings.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Same comment here about imposing VSCode settings.

Adding back the `.editorconfig` ensures that some of the settings you put in place here (ie space/tab size) are used across other IDEs and text editors.

The rest of the suggestions are subjective and we should let the user choose the settings they want.